{{cookiecutter.project_slug}}/requirements/local.txt

```diff
@@ -20,6 +20,10 @@ pytest-sugar==0.9.2  # https://github.com/Frozenball/pytest-sugar
 flake8==3.7.5  # https://github.com/PyCQA/flake8
 coverage==4.5.3  # https://github.com/nedbat/coveragepy
 black==19.3b0  # https://github.com/ambv/black
+pylint-django==0.9.4
```

Comment:
```suggestion
pylint-django==0.9.4  # https://github.com/PyCQA/pylint-django
```

Reply:
!!! thank you for your feedback and patience